tests/utils.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import hashlib
+from tempfile import TemporaryDirectory
+from zipfile import ZipFile
+from pathlib import Path
+
+files_directory = Path(__file__).parent / 'files'
+
+def setup_reference_fmus():
```

Comment:
I think we should rename this file to `conftest.py` and give this setup function the decorator 
`@pytest.fixture(autouse="True", session = "scope")`

Then it will be automatically run exactly ONCE with every `pytest` session.

See also: https://docs.pytest.org/en/6.2.x/fixture.html?highlight=conftest%20py#scope-sharing-fixtures-across-classes-modules-packages-or-session

Please have a look and see if you think this would be a good idea.

Reply:
I think it would be a great idea, thanks for sharing! I am not sure if we want `autouse=True`, it says:
_Sometimes you may want to have a fixture (or even several) that you know all your tests will depend on. “Autouse” fixtures are a convenient way to make all tests automatically request them_

I see the purpose of it but I think if we simply have the proposed fixture, without autouse,  on the setup-function, we should be good? Let me know what you think I don't have any strong opinions here.

Reply:
I see this creating the following issue:

One needs to use this decorator in every test that uses reference FMUs, which will likely be a few.

"autouse" gets around that. Having the decorator included on a file basis would be sensible I suppose, but we will likely be using the FMI3 reference FMUs in all FMI3 related tests sooner or later anyways.

Reply:
I see what you're saying. I have added the "autouse" now. I think also we are fine since we no longer download the zip if not necessary, so it should be minimal overhead.